torch_geometric/transforms/gdc.py

```diff
@@ -64,11 +68,20 @@ class GDC(BaseTransform):
 
     :rtype: :class:`torch_geometric.data.Data`
     """
-    def __init__(self, self_loop_weight=1, normalization_in='sym',
-                 normalization_out='col',
-                 diffusion_kwargs=dict(method='ppr', alpha=0.15),
-                 sparsification_kwargs=dict(method='threshold',
-                                            avg_degree=64), exact=True):
+    def __init__(
+        self,
+        self_loop_weight: float = 1.,
+        normalization_in: str = 'sym',
+        normalization_out: str = 'col',
+        diffusion_kwargs: Dict[str,
+                               Union[str,
+                                     float]] = dict(method='ppr', alpha=0.15),
+        sparsification_kwargs: Dict[str,
+                                    Union[str,
+                                          int]] = dict(method='threshold',
+                                                       avg_degree=64),
```

Comment:
Not sure`diffusion_kwargs` and  `sparsification_kwargs` are indeed necessary.

Reply:
Changed it to `Any`.